src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2370,6 +2377,12 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
         else:
             network_profile.network_dataplane = self.context.get_network_dataplane()
 
+        network_observability = self.context.get_enable_network_observability()
+        if network_observability is not None:
+            network_profile.network_monitoring = self.models.NetworkMonitoring(
```

Comment:
Queued a [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=74819996&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=a26fe913-e2be-5062-b08e-d15f1acc7ea2&l=576) and the create test case failed.

Seems the property name used here is wrong. Should be `network_profile.monitoring` instead of `network_profile.network_monitoring`.